logstash-offline/README.md

```diff
@@ -0,0 +1,79 @@
+## Getting Started
+
+### Instructions to register a project in Insightfinder.com
+- Go to the link https://app.insightfinder.com/
+- Sign in with the user credentials or sign up for a new account.
+- Go to Settings and Register for a project under "Insight Agent" tab.
+- Give a project name, select Project Type as "Log" and Agent Type as "Custom".
+- Note down the project name and license key which will be used for agent installation. The license key is also available in "User Account Information". To go to "User Account Information", click the userid on the top right corner.
+
+InsightFinder can parse log data sent via logstash. We utilize json_encode filter plugin and http-output plugin to format and send the data. 
+
+
+### 1. Requirements
+
+Ansible: Deploy and install logstash
+Filebeat: Streaming data to logstash with preconfigured fields for data parsing
+
+
+### 2. Configuration
+
+In the hosts file, several things need to be specified in order to run the playbook.
+
+In logstash-offline/hosts
+   Under [logstash], add hosts to this tag in the form of an FQDN or an IP address. 
+   Under [logstash:vars], the variables presented need to be filled out as such:
+      - username: the username of the InsightFinder user
+      - license_key: the key associated with the InsightFinder User
+      - server_url: the url used to reach InsightFinder. Use only the base url, for example: app.insightfinder.com
+      - logstash_user: the user to host the logstash processes
```

Comment:
If this is a duplicate of the ls_user, we should merge. Should keep the logstash_user and rename the ls_group to logstash_group to maintain consistency

Reply:
Done.